tasks/travis-pr-title-lint.js

```diff
@@ -0,0 +1,159 @@
+/*eslint prefer-template: 0*/
+
+var exec = require('child_process').exec;
+var https = require('https');
+
+var changelogLintPkg = require('conventional-changelog-lint');
+var objectValues = require('object.values');
+var objectEntries = require('object.entries');
+
+
+var changelogLint = changelogLintPkg.default;
+var getPreset = changelogLintPkg.getPreset;
+var getConfiguration = changelogLintPkg.getConfiguration;
+var format = changelogLintPkg.format;
+
+
+module.exports = function(grunt) {
+  function countGitMergeCommits() {
+    return new Promise(function(resolve, reject) {
+      exec('git rev-list --count HEAD ^master', function(err, stdout) {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(parseInt(stdout));
+        }
+      });
+    });
+  }
+
+  function getGitLastCommitMessage() {
+    return new Promise(function(resolve, reject) {
+      exec('git show -s --format=%B HEAD', function(err, stdout) {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(stdout);
+        }
+      });
+    });
+  }
+
+  function getPullRequestTitle() {
+    return new Promise(function(resolve, reject) {
+      var pullRequestURLPath = `/repos/${ process.env.TRAVIS_REPO_SLUG
+        }${/pulls/ }${process.env.TRAVIS_PULL_REQUEST }.json`;
```

Comment:
ouch, it shouldn't be there, I tried to remove any ES6 syntax to keep it "ES5"-safe :-)
but I forgot a bunch of them (but it looks like template strings are supported in node >= 4).

removed in the updated pull request.